scripts/graphics/gstshark-plot

```diff
@@ -20,6 +20,9 @@ then
     echo "                                 over each graphic generated"
     echo "                                 extern: display the legend over an external window"
     echo "                                 (default: inside)"
+    echo " -f, --filter [descriptor]       Filter events that have the word provided. For example"
+    echo "                                 Select one element information: $0 gstshark_filtered -p --filter x264enc"
+    echo "                                 Select more the one element:    $0 gstshark_filtered -p --filter \"x264enc|udpsink\""
```

Comment:
is this some standard regex syntax? if so, can we specify so?

Reply:
it Is a grep sintax, is an option to allow an "or" filter

Reply:
Ok I see you use `grep -E`, that’s extended regex. Please specify this, so users can know which syntax is valid

Reply:
Done